templates/bartholomew/content/static/example.txt

```diff
@@ -0,0 +1,4 @@
+This is an example of a static text file.
+
+If you can see this in your browser at `http://localhost:3000/static/example.txt`, then you have
```

Comment:
TODO: show the HTTP base here (but might need some magic to manage the right number of slashes in all circumstances).